goldstone/core/models.py

```diff
@@ -2634,3 +2633,38 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+class EventDef(DailyIndexDocType):
```

Comment:
EventDef should extend PolyResource rather than DailyIndexDocType.  The later represents a document type in ES.